android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/wallet/DialogWalletInstall.java

```diff
@@ -87,6 +96,16 @@ private void buildTop() {
       }
     });
 
+    hasImage = getContext().getResources()
```

Comment:
hasImage can be local
with this hasImage assignment, you can delete the previous assignment since it’s overriden by this